gallery/src/pages/StylingComponents.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const StylingRSCPage = () =>
+  <GalleryTile title="How to style your app when using RSC">
+    <p className="nx-p">
+      If you have styling needs that aren't covered by the styles included in React Shared Components (RSC),
+      then you're going to have to create your own styles.
+    </p>
+    <section className="nx-tile-subsection">
+      <header>
+        <h3 className="h3 nx-h3--tile-subsection">Page &amp; component specific stylesheets</h3>
+      </header>
+      <p className="nx-p">
+        Take a look at <code className="nx-code">/gallery/src/main.scss</code>. This file
+        contains custom styles for the Gallery and imports external stylesheets. Why have page and component
+        specific stylesheets? By grouping our styles together by use and (ideally) locating the SCSS
+        next to the HTML that it modifies we decrease clutter/noise in our main stylesheet, keep our styles focused
+        to what they modify, and also help maintain a clear separation between code, layout, and styling.
+      </p>
+      <h4 className="h4">Modifying component styles</h4>
+      <p>
+        It is not recommended that you directly modify any of the component styles in the RSC as a part of your
+        project because your changes are likely to be overwritten the next time you update the RSC in your project.
```

Comment:
Is this referring to modifying the scss files that get installed as part of RSC in `node_modules`?  I would hope it goes without saying that modifications there are not a valid approach, to anything (css, js, etc).

Reply:
> Is this referring to modifying the scss files that get installed as part of RSC in `node_modules`? I would hope it goes without saying that modifications there are not a valid approach, to anything (css, js, etc).

I would also hope that it goes without saying, but I'll bet it isn't, that's why I called it out.

Reply:
> I would also hope that it goes without saying, but I'll bet it isn't, that's why I called it out.

Keep in mind that such a strategy wouldn't even work, in the face of CI-based builds and a multi-person team, since the `node_modules` folder doesn't get committed.

Reply:
> > I would also hope that it goes without saying, but I'll bet it isn't, that's why I called it out.
> 
> Keep in mind that such a strategy wouldn't even work, in the face of CI-based builds and a multi-person team, since the `node_modules` folder doesn't get committed.

True, best to not even suggest the possibility: https://github.com/sonatype/sonatype-react-shared-components/pull/72/commits/a4f3abe65e574df6ec54fd1948361376a634ef64